src/views/layouts/main.hbs

```diff
@@ -3,6 +3,11 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content ="width=device-width, initital-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Architects+Daughter|Mali" rel="stylesheet">
```

Comment:
You've linked to this font but you're not using it, I would add this to styles or it's not necessary here.